pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -3060,7 +3059,7 @@ var _ = Describe("Converter", func() {
 		)
 
 		BeforeEach(func() {
-			vmi = kvapi.NewMinimalVMI("testvmi")
+			vmi = libvmi.New(libvmi.WithName("testvmi"))
 			v1.SetObjectDefaults_VirtualMachineInstance(vmi)
 			vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
 			vmi.Spec.Domain.Devices.AutoattachMemBalloon = pointer.P(true)
```

Comment:
You can use `libvmi.WithRng()` and also create a new option for `AutoattachMemBalloon`